profiles/community/pulp_config.env

```diff
@@ -0,0 +1,49 @@
+PULP_CONTENT_PATH_PREFIX=/api/v3/artifacts/collections/
+
+PULP_GALAXY_API_PATH_PREFIX=/api/
+PULP_GALAXY_AUTHENTICATION_CLASSES=['rest_framework.authentication.SessionAuthentication', 'rest_framework.authentication.TokenAuthentication', 'rest_framework.authentication.BasicAuthentication', 'django.contrib.auth.backends.ModelBackend']
+PULP_GALAXY_DEPLOYMENT_MODE=standalone
+PULP_GALAXY_REQUIRE_CONTENT_APPROVAL=false
+PULP_GALAXY_AUTO_SIGN_COLLECTIONS=false
+PULP_RH_ENTITLEMENT_REQUIRED=insights
+
+PULP_ANSIBLE_API_HOSTNAME=http://localhost:5001
```

Comment:
Don't duplicate settings in galaxy_ng/base. These should be inherited from the base profile.